darwin/importer/formats/__init__.py

```diff
@@ -2,9 +2,10 @@
 
 from darwin.datatypes import ImporterFormat
 
-from . import coco, csvtags, csvtagsvideo, darwin, dataloop, pascalvoc
+from . import coco, csvtags, csvtagsvideo, darwin, dataloop, label_box, pascalvoc
```

Comment:
this is really really unimportant, but I think we should call that file `labelbox` (no underscore, like the actual startup name) for consistency with other files (`data loop`, `pascalvoc`, etc)

Reply:
done !